array-checker.js

```diff
@@ -0,0 +1,13 @@
+function uniqueCalc(arr1, arr2) {
+  let unique1 = arr1.filter((o) => arr2.indexOf(o) === -1);
+  let unique2 = arr2.filter((o) => arr1.indexOf(o) === -1);
```

Comment:
`unique1` and `unique2` variables don't get re-assigned, so you should better use `const` instead of `let` here.